books/free-programming-books-ru.md

```diff
@@ -134,7 +134,7 @@
 ### C
 
 * [Заметки о языке программирования Си/Си++](https://yurichev.com/writings/C-notes-ru.pdf) - Денис Юричев (PDF)
-* [Краткое руководство Beej к GDB](https://paintingvalley.com/ru-bggdb) - (HTML)
+* [Краткое руководство Beej к GDB](https://paintingvalley.com/ru-bggdb) - Brian "Beej Jorgensen" Hall (HTML)
```

Comment:
(note: it wasn't clear to me who the translation was provided by for this existing entry; I considered attempting to add a credit along with the name change)